src/peer_manager.rs

```diff
@@ -284,7 +284,7 @@ impl PeerManager {
     pub fn reset_routing_table(&mut self, our_public_id: PublicId, groups: &[Group]) {
         self.unknown_peers.clear();
         self.expected_peers.clear();
-        let min_group_size = self.routing_table.min_group_size();
+        let min_group_size = self.routing_table.min_size();
```

Comment:
I'd call that `min_section_size` or `merge_size` or leave it as `min_group_size` for now. To me, `min_size` sounds like it refers to the whole routing table.

(I also wonder whether in general we should use `len` instead of `size`, as that's what the standard library does.)